src/api-handlers/candidates.js

```diff
@@ -90,12 +91,61 @@ async function getCandidates(fastify, request) {
 
 }
 
+async function getPlanetCandidates(fastify, request) {
+    const {
+        query: { walletId },
+        params: { dacId },
+    } = request;
+    const config = loadConfig();
+    const logger = require('../connections/logger')('candidates', config.logger);
+    const dacConfig = await loadDacConfig(fastify, dacId);
+    const api = new Api({
+        rpc: new JsonRpc(config.eos.endpoint, {fetch}),
+        signatureProvider: null,
+        chainId: config.chainId,
+        textDecoder: new TextDecoder(),
+        textEncoder: new TextEncoder(),
+    });
+    const db = fastify.mongo.db;
+    //
+    const candidates = await getCandidates(logger, api, dacId, 100);
+    const votedCandidates = await getVotedCandidates(logger, api, dacId, walletId);
+
+    if (candidates.length === 0) {
+        return [];
+    }
+
+    const profiles = await getCandidatesProfiles(
+        logger,
+        db,
+        dacConfig,
+        dacId,
+        candidates.map(candidate => candidate.candidate_name),
+    );
+    const terms = await getMemberTerms(logger, api, dacId, 1);
```

Comment:
Probably the same applies here, https://github.com/Alien-Worlds/eosdac-api/pull/25#discussion_r1010180178